README.md

```diff
@@ -168,6 +186,10 @@ Below are some commonly encountered issues and possible ways to resolve them. If
 - The app gets stuck on the Projects screen -- projects never load
   > Make sure the API is running on your local machine, and that your **api/.env** and **app/.env** files are configured correctly (see [Setup and first run](#setup-and-first-run) above)
 
+  > Check the URL for both `API_TUNNEL_SUBDOMAIN` and `STA_API_BASE_URL`. `STA_API_BASE_URL` should be prefixed with `https://`. `API_TUNNEL_SUBDOMAIN` should *not* be prefixed with `https://` and should *only* include the URL up to but **not** including `.loca.lt`.
```

Comment:
`API_TUNNEL_SUBDOMAIN` isn't a URL, it's just a string (e.g. `nadia-bloggs-abc123def456ghi789`) so please can you change this to "Check the values for both"

Reply:
Good point - i have amended the start of this sentence top reflect this.